charts/ark-samba/values.yaml

```diff
@@ -2,52 +2,62 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
-replicaCount: 1
-
 # This can be set to "false" by parent charts. If not defined, it will be taken as true
 #enabled: true
 
-global:
-  imageRegistry: ""
-  ## E.g.
-  ## imagePullSecrets:
-  ##   - myRegistryKeySecretName
-  ##
-  imagePullSecrets: []
-  storageClass: ""
+#
+# This should group all application-specific configurations
+#
+configuration:
+  domain:
+    name: "arkcase.test"
+    password: "RaL@a5eio:heC3"
```

Comment:
Should we take the passwords from a vault, or from the environment?

Reply:
This is something I've been pondering on how to properly handle when Helming ... the base intention is to put them in a configuration which then gets stuffed into secrets, and consumed by pods from there as required ... but I'm not happy with putting credentials in cleartext in a values file for the deployment.  That, however, is still a bit of a ways away since I need to get this working first ...

Reply:
This is probably the way: https://learn.hashicorp.com/tutorials/vault/kubernetes-sidecar

Need to learn how to do it, though ...